apps/filecoin-site/src/app/blog/components/BlogCard.tsx

```diff
@@ -0,0 +1,63 @@
+import Link from 'next/link'
+
+import {
+  CardImage,
+  type CardImageProps,
+} from '@filecoin-foundation/ui/Card/CardImage'
+
+import { Heading } from '@/components/Heading'
+import { TagGroup } from '@/components/TagGroup/TagGroup'
+
+import { type BlogPost } from '../schemas/BlogPostFrontmatterSchema'
+
+import { PostMetadata } from './PostMetadata'
+
+type BlogCardLink = {
+  href: string
+  ariaLabel: string
+}
+
+export type BlogCardProps = {
+  title: BlogPost['title']
+  cta?: BlogCardLink
+  tags?: BlogPost['categories']
+  description?: BlogPost['excerpt']
+  image?: CardImageProps
+  author?: BlogPost['author']
+  date: BlogPost['date']
+}
+
+export function BlogCard({
+  title,
+  tags,
+  date,
+  description,
+  cta,
+  image,
+  author,
+}: BlogCardProps) {
+  return (
+    <article aria-label={title} title={title} className="group relative h-full">
+      {image && <CardImage image={image} />}
+
+      <div className="flex flex-col gap-4 py-4">
+        {tags && <TagGroup tags={tags} />}
+        <Heading tag="h3" variant="xl-medium" className="leading-tight">
```

Comment:
Let's leave the `leading-tight` out for now and take note to add leading styles to the `Heading` component.

Reply:
👍 Added the note [here](https://www.notion.so/filecoin/filecoin-io-V3-Followup-Dev-Work-22a7631f282580abb259db9f137b481c?source=copy_link)